flake8_vedro/visitors/context_checkers/assert_checker.py

```diff
@@ -0,0 +1,38 @@
+import ast
+from typing import List
+
+from flake8_plugin_utils import Error
+
+from flake8_vedro.abstract_checkers import ContextChecker
+from flake8_vedro.errors import ContextWithoutAssert
+from flake8_vedro.visitors.context_visitor import Context, ContextVisitor
+
+
+@ContextVisitor.register_context_checker
+class ContextAssertChecker(ContextChecker):
+
+    def check_context(self, context: Context, config) -> List[Error]:
+        errors = []
+
+        has_assert = False
+        for line in context.context_node.body:
+            if isinstance(line, ast.Assert):
+                has_assert = True
+                break
+
+            has_assert = self._has_assert_in_block(context.context_node.body)
```

Comment:
you have a loop **for every line in context.context_node.body**, for each line there's a call of method _has_assert_in_block for **the whole context.context_node.body**

are you sure that it's a good solution?